src/tests/handlers/breeds-get.test.ts

```diff
@@ -0,0 +1,88 @@
+import fetch from 'node-fetch'
+import { getBreedsHandler, getRandomBreedHandler } from '../../handlers/breeds.handler'
+import { BreedsResponse } from '../../domain/breed.domain'
+import context = require('aws-lambda-mock-context')
```

Comment:
There should be a way to import this module without mixing `import` and `require`

Reply:
I have mixed this and use thos syntax to avoid lint require import error.

Reply:
Valid solution, though this module should still be able to be imported via `import` to maintain consistency. It might require using `import * as ...` syntax, depends on the package.

Reply:
yes, consistency is always better and preferable

Reply:
updated code.